Пак СССП

AltMapper opened the pull request:
Вооружение ВС СССП

<!-- ЭТО ШАБЛОН ВАШЕГО PULL REQUEST. Текст между стрелками - это комментарии - они не будут видны в PR. -->

## Описание PR
<!-- Ниже опишите ваш Pull Request. Что он изменяет? На что еще это может повлиять? Постарайтесь описать все внесённые вами изменения! --> Добавлена лазерная винтовка Вооруженных сил СССП - АБЗЛ и основная автоматическая винтовка ВС СССП - АБЗ-84.

## Цель / Баланс
<!-- Зачем было сделано изменение? Укажите ссылки на любые задачи или дискуссии в этом пункте. Пожалуйста укажите, как это повлияет на игровой баланс. --> Оружие добавлено для проведение лорных ивентов, а также щитспавна.

## Медиа
<!-- Если приемлемо, добавьте скриншоты для демонстрации вашего PR. Если ваш PR представляет собой визуальное изменение, добавьте
скриншоты, иначе он может быть закрыт. -->
- [x] Я добавил скриншот/видео демонстрации PR в игре, **или** этот PR этого не требует. 

![изображение_2024-05-27_170330011](https://github.com/ss14-ganimed/ENT14-Master/assets/141431347/170bfc7f-cfac-4634-a2a0-dfe2f4e43690)
![image](https://github.com/ss14-ganimed/ENT14-Master/assets/141431347/1da9b9c2-01c1-4f73-a97c-99acf8d9f802)
![image](https://github.com/ss14-ganimed/ENT14-Master/assets/141431347/da30e2a3-7753-4317-83e6-ac2040677a29)

-->

## Проверки
<!-- Выполнение всех следующих действий, если это приемлемо для вида изменений сильно ускорит разбор вашего PR
[X] пометит галочкой -->
- [x] PR полностью завершён и мне не нужна помощь чтобы его закончить.
- [x] Я внимательно просмотрел все свои изменения и багов в них не нашёл.
- [x] Я запускал локальный сервер со своими изменениями и всё протестировал.

## Изменения
<!--
Убедитесь, что игроки остануться в курсе новых изменений, добавив запись в список изменений.

В журнал изменений следует помещать только то, что действительно важно игрокам.

В списке изменений тип значка не является часть предложения, поэтому явно указывайте - Добавлен, Удалён, Изменён.
плохо: - add: Новый инструмент для инженеров
хорошо: - add: Добавлен новый инструмент для инженеров

Вы можете указать своё имя после символа :cl: именно оно будет отображаться в журнале изменений (иначе будет использоваться ваше имя на GitHub)
Например: :cl: Ian

-->

:cl: <!-- Вставьте сюда ваш никнейм! -->
- add:  Добавлены АБЗ-84 (автоматическая винтовка), АБЗЛ (лазерная винтовка)
- fix: АКМС, в соответствии со спрайтом, переименован в АКМ

AltMapper committed:
Пак оружия СССП

Вооружение ВС СССП

AltMapper committed:
Исправление ошибки в мете Мультифазного пистолета.

Пока делал АБЗЛ случайно удалил один из файлов в meta-файле Х-01. Заметил и исправил ошибку

Signed-off-by: AltMapper <141431347+AltMapper@users.noreply.github.com>

Gorox221 reviewed (approved):
Берем.

Gorox221 reviewed (approved):
Почему-то не апрувнулось. Попытка номер 2

AltMapper committed:
Завершение добавления вещей из пака СССП

Перенесён весь собранный пак СССП, включающий: берет, знамя, два вида униформы и два вида скафандров.

AltMapper committed:
Merge branch 'USSPWeapons' of https://github.com/AltMapper/ENT14-Master into USSPWeapons

AltMapper committed:
Удалил какой-то лишний коммит

Signed-off-by: AltMapper <141431347+AltMapper@users.noreply.github.com>

AltMapper committed:
Revert "Удалил какой-то лишний коммит"

This reverts commit 1d6caaff13e908fc4e0c665a2079d3896e7fdf2a.

AltMapper committed:
Гарнитура и частота СССП

Добавлены гарнитура, ключ-шифрования и частота СССП.
